docs/typescript.md

```diff
@@ -89,68 +89,70 @@ import React from 'react';
 import {Button, StyleSheet, Text, View} from 'react-native';
 
 export interface Props {
-    name: string;
-    enthusiasmLevel?: number;
```

Comment:
I'm also curious - how did you update the unrelated whitespace changes?

Did you do these manually or with a script?

Reply:
Hello, @rickhanlonii, I think this is something my editor has 'helpfully' done for me - I'll take a look and get the tests green then resubmit.

Reply:
Ah yeah it may be a default prettier config that runs on markdown files